plugins/mc.yaml

```
@@ -0,0 +1,64 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: mc
+spec:
+  version: v0.5.0
+  homepage: https://github.com/jonnylangefeld/kubectl-mc
+  shortDescription: Run kubectl commands against multiple clusters at once
+  description: |+2
+    A Multi Cluster kubectl Client
+
+    If you work at a company or organization that maintains multiple Kubernetes
+    clusters it is fairly common to connect to multiple different kubernetes
+    clusters throughout your day. And sometimes you want to execute a command
+    against multiple clusters at once. For instance to get the status of a
+    deployment across all `staging` clusters.
```

More users will read this, if the text is short. A short description takes longer to write, but it is worth it!

How about:

```suggestion
    This plugin helps to work with multiple kubernetes clusters at once. For instance
    to get the status of a deployment across all `staging` clusters.
```